frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -320,6 +324,7 @@ function VirtualKey:onHoldSelect()
         -- Don't refresh the key region if we're going to show a popup on top of it ;).
         if self.hold_callback then
             self[1].inner_bordersize = 0
+            self:update_keyboard(false, true)
```

Comment:
Just requesting @NiLuJe confirmation on these flashy stuff :)

Reply:
Without it the border remains black after hold.
Popup small additional keyboards works fine with it.

Reply:
I'd rather fix it in the problematic cb rather than unconditionally ;).